### PR TITLE
feat(ibmmq): clarify required persmissions

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Updating ibmmq permissions specifications https://new-relic.atlassian.net/browse/NR-259871, https://new-relic.atlassian.net/browse/NR-262491

### CODE REVIEW

**Comment by homelessbirds** [comment]
@paologallinaharbur - thanks for this! It's a little unclear to me which permissions need to be added. It lists the capabilities but not what permissions need to be granted. Am I missing something?

**Comment by paologallinaharbur** [comment]
No, I think it is just my "English" having some issues 😄 
Those I've listed are the permissions/capabilities needed. I am not really sure about the difference. 

I though that "being able to access a QM" or "being able to access a specific channel" was considered a `permission`.
The ibmmq docs refers to them as "authorizations", but the UI as permissions
image

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ibmmq-monitoring-integration.mdx [hunk 1]
```diff
@@ -140,6 +139,17 @@ To configure the integration, edit the config in the integration's YAML config f
...
 
+### Permissions [#permissions]
+The `username` set in the integration config needs to be granted some permissions in order to monitor a IBM MQ infrastructure.
+The required permissions varies depending on monitored queues, channels, and topics.
+
+In general, the user should be able to:
+ - connect to the queue manager having `DISPLAY` and `INQUIRY` enabled.
+ - access the channel specified in the integration config (by default `SYSTEM.DEF.SVRCONN`)
+ - operate priviledge commands on queues `SYSTEM.DEFAULT.MODEL.QUEUE` and `SYSTEM.ADMIN.COMMAND.QUEUE` having the `Display` permission.
+ - access with `Display` and `Subscribe` for the topics `SYSTEM.ADMIN.TOPIC` and `SYSTEM.BROKER.ADMIN.STREAM`
```
Comment (paologallinaharbur) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/ibmmq-monitoring-integration.mdx:150]: these are the permissions required, however some additional one are needed depending on their specific setup

File: src/content/docs/infrastructure/host-integrations/host-integrations-list/ibmmq-monitoring-integration.mdx [hunk 2]
```diff
@@ -140,6 +139,16 @@ To configure the integration, edit the config in the integration's YAML config f
 
 To read all about these common settings, refer to our [configuration format](/docs/create-integrations/infrastructure-integrations-sdk/specifications/host-integrations-newer-configuration-format/#configuration-basics) document.
 
+### Permissions [#permissions]
+To monitor IBM MQ infrastructure, the `username` designated in the integration config must be granted certain permissions. These permissions vary depending on monitored queues, channels, and topics.
+
+In general, the `username` should have the following permission:
```
Comment (homelessbirds) [inline src/content/docs/infrastructure/host-integrations/host-integrations-list/ibmmq-monitoring-integration.mdx:145]:
```suggestion
In general, the `username` should have the following permissions: 
```

**Review by homelessbirds — APPROVED** [review]

**Pull request merged**